reference/examples/LazyCreateAccount.md

```diff
@@ -0,0 +1,18 @@
+Lazy-create a new account using a public-address via the CryptoCreate transaction.
+
+Reference: HIP-583
+
+## Example 1
+- Create an Ethereum account address/public-address (example: 0xb794f5ea0ba39494ce839613fffba74279579268)
+- Use the `AccountCreateTransaction` to create an account using the Ethereum account address/public address
+  - Use the `setAliasEVMAddress()` 
+  - Do not use the `setKey` field or `setAlias()` field
+- Sign with... // what key? The testnet account ID and key or the private key of the Ethereum account public-address?
```

Comment:
What account/key is signing the AccountCreateTransaction?

Reply:
Account creation for hollow accounts can be signed by any one. Just as you would for a normal crypto create. It's the follow up transaction to complete an account that should have the public